proto/batch_plan.proto

```diff
@@ -198,9 +198,10 @@ message LookupJoinNode {
   expr.ExprNode condition = 2;
   repeated int32 build_side_key = 3;
   plan_common.CellBasedTableDesc probe_side_table_desc = 4;
-  repeated int32 probe_side_column_ids = 5;
-  repeated uint32 output_indices = 6;
-  repeated ExchangeSource sources = 7;
+  repeated uint32 probe_side_vnode_mapping = 5;
+  repeated int32 probe_side_column_ids = 6;
+  repeated uint32 output_indices = 7;
+  repeated common.WorkerNode worker_nodes = 8;
```

Comment:
Rename it to `all_worker_nodes`?

Reply:
How about putting the `worker_nodes` and vnode_mapping in `BatchTaskContext`? I think putting them in plan node maybe too huge,while putting them `BatchTaskContext`, we just need to maintain on e copy for each compute node, and in frontend, it already exists.

Reply:
I think this would be more complicated to implement and it also seems unnecessary since only `LookupJoinExecutor` is using the `vnode_mapping` and `worker_nodes`.